autopts/wid/gatt_client.py

```diff
@@ -1289,6 +1289,48 @@ def hdl_wid_109(params: WIDParams):
     return True
 
 
+def hdl_wid_130(params: WIDParams):
+    """
+    Please delete security key before connecting to PTS if IUT was bonded previously.
+    """
+    btp.gap_unpair()
+
+    return True
+
+
+def hdl_wid_135(params: WIDParams):
+    """
+    Please write to client support feature handle = 'XXXX'O to enable Robust Caching.
+    Discover all characteristics if needed.
+    """
+
+    pattern = re.compile(r"'([0-9a-fA-F]+)'")
+    params = pattern.findall(params.description)
+    if not params:
+        logging.error("parsing error")
+        return False
+
+    handle = params[0]
+
+    # First read the existing value in Client Supported Features.
+    btp.gatt_cl_read(btp.pts_addr_type_get(), btp.pts_addr_get(), handle)
+
+    stack = get_stack()
+    stack.gatt_cl.wait_for_verify_values()
+    client_support_features = btp.get_verify_values()
+
+    # Set Robust Caching bit in features.
+    robust_caching_bit = 1
+    value = int(client_support_features[1]) | robust_caching_bit
```

Comment:
Let's try to avoid magic numbers
Suggest to define the `client_support_features` bits and use define values. 
Additionally, why is this only setting `client_support_features[1]` and not both `[0]` and `[1]`?